easy_tst/tst_wrapper.py

```diff
@@ -197,6 +113,28 @@ def request_to_tst():
     return response
 
 
-# Check if a file is empty
-def is_zero_file(fpath):
-    return not (os.path.isfile(fpath) and os.path.getsize(fpath) > 0)
+def cache_atualizer():
+    response = request_to_tst()
+
+    if response.status_code == 400:
+        raise RuntimeError('Not logged in tst.')
+
+    return response.json()
+
+
+def main(code):
+    cache = cache_atualizer()
```

Comment:
Aqui a cache vai ser sempre atualizada quando for chamado o wrapper. Checa se ela ta vazia, se sim, `update_cache()`. Se nao, ela mesmo. Lembra daquele `CACHE` que tinha no clipboard.

Ai la no checkout (linha 36) ve se o exercicio ta presente. Se nao tiver, vai ser necessario atualizar a cache, pois pode ser um exercicio novo

Reply:
Ai voltaria a ter aquela variável global?

Reply:
No momento não vejo como resolver sem ser global nesse ponto. Senão vamos voltar àquele problema de perfomance (um request por checkout)

Reply:
Ok